wiki/markdown/running-rules-on-objects.md

```diff
@@ -33,15 +33,15 @@ A GUI is also implemented in [Eiffel Intelligence Frontend](https://github.com/e
 
 Using this arbitrary endpoint one can use check if testRules are enabled in this EI instance.
```

Comment:
You can change this sentence to something like this:
Using the root endpoint one can check if rule tests are enabled in the EI instance.

Reply:
The sentence I thought was a bit weird was the one above the /rule-test command.

Reply:
Ye it was my fault, i mixed it up :D